src/components/MovieDetails.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../styles/MovieDetails.css';
+
+const MovieDetails = ({ selectedMovie, returnHome }) => {
+
+  return (
+    <div>
+      <div className="movie-details-container">
+        <h2 className="title">{selectedMovie.title} ({selectedMovie.release_date})</h2>
+          <img className="movieBackDrop" src= {selectedMovie.backdrop_path} alt={selectedMovie.overview}/>
```

Comment:
Great job putting in alt text. Do you think having the full description as the alt text is the best? Alt text should describe the image itself, so maybe something like "movie poster for <movie name>" would be better.